### PR TITLE
aih employee name

### PR DESCRIPTION
**Pull request by SidelineCory24**
[BE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/682)
To-do:
- [x] add employee name to header
- [x] employee name should link to the employee profile for CDO
- [x] employee name should not link to the employee profile for AO
- [x] disable the ability to edit the TED, TOD, Action, & Travel rows in the AIH View

### CODE REVIEW

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx [hunk 1]
```diff
@@ -97,6 +98,7 @@ AgendaItemRow.propTypes = {
   showEdit: PropTypes.bool,
   isCDO: PropTypes.bool,
   perdet: PropTypes.number,
+  isAIHView: PropTypes.bool,
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:101]: nice catch 😅

**Comment by elizabeth-jimenez** [comment]
in file `src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx` line 56, you can just pass in client `agendaItemHistoryExport(id, sort, client)`

**Comment by elizabeth-jimenez** [comment]
Might want to also update the `fileNameAlt` line 31 in `src/actions/agendaItemHistory.js` to reflect BE updates
image

File: src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx [hunk 2]
```diff
@@ -29,7 +31,8 @@ const AgendaItemHistory = (props) => {
   const [sort, setSort] = useState(sorts.defaultSort);
 
   // To-do - get client name. Can't get person using their perdet
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx:33]: 🧼 ?

File: src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx [hunk 3]
```diff
@@ -71,17 +74,36 @@ const AgendaItemHistory = (props) => {
...
-  if (client) {
-    title = `${client}'s ${title}`;
-  }
+  const title = 'Agenda Item History';
 
   const exportDisabled = (aih || []).length <= 0;
 
   return (
     <div className="agenda-item-history-container">
+      <div className="aih-header-container">
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx:83]: there is nothing between `aih-header-container` and `aih-title-container`, so we can remove 1 of the 2 and combine the stylings.

File: src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx [hunk 4]
```diff
@@ -71,17 +74,36 @@ const AgendaItemHistory = (props) => {
   }, [sort]);
 
   const isLoading$ = isLoading;
-  let title = 'Agenda Item History';
-  if (client) {
-    title = `${client}'s ${title}`;
-  }
+  const title = 'Agenda Item History';
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx:77]: doesn't have to be a variable, since we only use it once

**Comment by elizabeth-jimenez** [comment]
RIP
image

File: src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx (hunk 2 above)
Comment (SidelineCory24) [reply src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx:33]: done

File: src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx (hunk 4 above)
Comment (SidelineCory24) [reply src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx:77]: done

File: src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx (hunk 3 above)
Comment (SidelineCory24) [reply src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx:83]: done

**Comment by SidelineCory24** [comment]
> Might want to also update the `fileNameAlt` line 31 in `src/actions/agendaItemHistory.js` to reflect BE updates image

done

**Comment by SidelineCory24** [comment]
> in file `src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx` line 56, you can just pass in client `agendaItemHistoryExport(id, sort, client)`

changes made on FE and BE

File: src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx [hunk 5]
```diff
@@ -127,6 +145,7 @@ const AgendaItemHistory = (props) => {
                         isCreate
                         isCDO={isCDO}
                         perdet={id}
+                        isAIHView
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx:148]: don't need here

File: src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx [hunk 6]
```diff
@@ -149,6 +169,7 @@ const AgendaItemHistory = (props) => {
...
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx:172]: dont need here

**Comment by elizabeth-jimenez** [comment]
Just a few comments, but good to go otherwise

File: src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx (hunk 5 above)
Comment (SidelineCory24) [reply src/Components/Agenda/AgendaItemHistory/AgendaItemHistory.jsx:148]: done

**Comment by SidelineCory24** [comment]
> Just a few comments, but good to go otherwise

requested changes made for export

**Review by elizabeth-jimenez — APPROVED** [review]

**Pull request merged**